pkg/agent/core/ngt/handler/grpc/index_test.go

```diff
@@ -508,106 +508,529 @@ func Test_server_CreateIndex(t *testing.T) {
 func Test_server_SaveIndex(t *testing.T) {
 	t.Parallel()
 	type args struct {
-		ctx context.Context
 		in1 *payload.Empty
```

Comment:
could you update this value to more understandable?

Reply:
I am sorry that I cant understand what you mean?

Reply:
fixed by rename it to `in`